f8a_worker/solver.py

```diff
@@ -307,9 +307,18 @@ def fetch_releases(self, package):
         if not package:
             raise ValueError('package not specified')
 
-        gh_host, gh_org, gh_proj = package.split("/")[:3]
-        repo_url = 'git://{gh_host}/{gh_org}/{gh_proj}.git'.format(
-            gh_host=gh_host, gh_org=gh_org, gh_proj=gh_proj)
+        print("Input package: %s" % package)
```

Comment:
Looks like left-over debug message.
You can use  `logger.debug()` if you need it, but make the message more descriptive.